python/cog/server/runner.py

```diff
@@ -124,9 +139,13 @@ def is_busy(self) -> bool:
     def cancel(self, prediction_id: str) -> None:
```

Comment:
I wonder if we could consistently call the ID the "task ID"? Whether or not it's also the prediction ID, it feels like we're using it to identify the task, not the prediction, aren't we?